docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,85 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class in which each side of children can be specified as relative to a sibling view or parent. The `Target` and the `RelativeOffset` properties define a relationship with each other.
+
+The following figure shows the relative layout position of rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing:
+
+**Relative layout example with `Target` and `RelativeOffset`**
+
+![relativeLayoutExamplePreview](./media/relativeLayoutExamplePreview.png)
+
+```xaml
+<View x:Name="Parent">
+    <View.Layout>
+        <RelativeLayout/>
+    </View.Layout>
+
+    <View x:Name="RedRect"   BackgroundColor="1,0,0,1" WidthSpecification="100" HeightSpecification="100"/>
+    <View x:Name="GreenRect"  BackgroundColor="0,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="RedRect" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"
+          RelativeLayout.LeftRelativeOffset="1.0"/>
+    <View x:Name="BlueRect"  BackgroundColor="0,0,1,1" WidthSpecification="100" HeightSpecification="100"
+          RelativeLayout.RightRelativeOffset="1.0" RelativeLayout.LeftRelativeOffset="1.0"
+          RelativeLayout.HorizontalAlignment="End"/>
+    <View x:Name="YellowRect"  BackgroundColor="1,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="Parent" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"/>
+</View>
+```
+
+- When the `Target` and the `RelativeOffset` properties are set as default, the child `RedRect` is aligned to the left side of the parent layout.
+- The `BlueRect` is aligned to the right side of the parent layout by the `HorizontalAlignment`. 
+- The left side of `GreenRect` is set to be aligned with `RedRect` and the right side is set to be aligned with `BlueRect`. The `GreenRect` fills the space between `RedRect` and `BlueRect` by using `FillHorizontal`.
+- The `YellowRect` represents that each side can be set as sibling view and parent layout.
+
+**Relative layout example with `Target` and `RelativeOffset` on different screen resolutions**
+
+![relativeLayoutExamplePreviewDifferentScreenResolution](./media/relativeLayoutExamplePreviewDifferentScreenResolution.png)
+
+Since the children of relative layout are laid out depending on their relationship, you can create a responsive UI that keeps the proportions across different screen resolutions without modifications.
```

Comment:
```suggestion
Since the children of the relative layout are laid out depending on their relationship, therefore you can create a responsive UI that keeps the proportions across different screen resolutions without modifications.

Reply:
Done.